apps/api/src/expenses/dto/create-expense.dto.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  IsEnum,
+  IsNumber,
+  IsOptional,
+  IsString,
+  IsUUID,
+  MaxLength,
+  MinLength,
+} from 'class-validator'
+import { ApiProperty } from '@nestjs/swagger'
+import { Expose } from 'class-transformer'
+import { Currency, ExpenseType, ExpenseStatus } from '.prisma/client'
+
+@Expose()
+export class CreateExpenseDto {
+  @ApiProperty()
+  @Expose()
+  @IsString()
+  type: ExpenseType
+
+  @ApiProperty()
+  @Expose()
+  @IsString()
+  @MinLength(3)
+  @MaxLength(3)
+  @IsEnum(Currency)
+  currency: Currency
+
+  @ApiProperty()
+  @Expose()
+  @IsNumber()
+  amount: number
+
+  @ApiProperty()
+  @Expose()
+  @IsString()
```

Comment:
We can add better validation here by saying it's an enum - this will make the enum values the only correct values, not open to any string values as it is right now